lib/load-session-from-socket.js

```diff
@@ -43,7 +43,14 @@ module.exports = function loadSessionFromSocket(socketHandshake, app, cb) {
   if (!socketHandshake.headers.cookie || !cookieHasSailsSid){
     hadToGenerateCookie = true;
     oneTimeCookie = app.session.generateNewSidCookie();
-    socketHandshake.headers.cookie = [socketHandshake.headers.cookie, oneTimeCookie].filter(Boolean).join('; ');
+
+    // If non-sid cookie exists then just append the one-time cookie
+    if (socketHandshake.headers.cookie) {
```

Comment:
@mikermcneil I have updated this as per your suggestion. Should be good to go now!